sublime_linter.py

```diff
@@ -357,56 +377,73 @@ def group_by_filename_and_update(window, bid, view_has_changed, linter, errors):
     did_update_main_view = False
     for filename, errors in grouped.items():
         if not filename:  # backwards compatibility
-            update_buffer_errors(bid, linter, errors)
+            update_file_errors(main_filename, linter, errors)
         else:
-            # search for an open view for this file to get a bid
-            view = window.find_open_file(filename)
+            # search for an open view for this file
+            view = main_view.window().find_open_file(filename)
```

Comment:
Don't forget `window()` is one of those live getters and can return None and in fact while testing I already had a None exception here.

Reply:
If you look at the signature we had `Wid, Bid, ...` and it looks like we have `Wid, Filename, ...` right now.